docs/terminal/tindal.md

```diff
@@ -4,7 +4,97 @@
 
 --8<-- "includes/abbreviations.md"
 
-Reserved.
+## Positions
+
+| Name               | Callsign       | Frequency        | Login ID              |
+| ------------------ | -------------- | ---------------- | --------------------------------------|
+| **Tindal Approach**    | **Tindal Approach**   | **120.95** | **TN_APP** |
+
+## Airspace
+TN APP being online will activate the following Airspace by default:
+
+- TN MIL CTR `SFC`-`A025`
+- R249A `A015`-`A025`
+- R249B `A025`-`F190`
+- R238 `A035`-`F190`
+
+All of which are reclassified as **Class C** when active, and TN APP has Control Authority of.
 
 !!! note
-    Additional charts to the AIP may be found in the RAAF TERMA document, available towards the bottom of [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
\ No newline at end of file
+    TN APP may increase the upper limit of the Restricted Areas and their airspace. This must be either published by NOTAM, or negotiated with TRT(TRS) if online.
+
+## Charts
+!!! note
+    Additional charts to the AIP may be found in the RAAF TERMA document, available towards the bottom of [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
+
+## Arrival/Departure Gates
+There are 11 gates to facilate departues and arrivals from the TN Training Airspace, based on the TDL TACAN.  
+Departing aircraft will be cleared to track direct to the gate, or allowed to intercept the outbound radial by 10 DME.  
+Arrivals can be cleared via the appropriate gate for their return to YPTN.
+
+| Gate Name | Ident | Bearing (°M) | Distance (nm) | Restricted Area |
+| ------ | ----------| ----------| ----------| ----------|
+| BIAK     | BIA | 320 | 30 | N/A |
+| TOP GATE     | TOP | 300 | 30 | R225D |
+| MOROTAI     | MOR | 270 | 30 | R225D |
+| TADJI     | TAD | 240 | 30 | R225D |
+| TARAKAN     | TAR | 200 | 30 | R225D |
+| BOTTOM GATE     | BOM | 190 | 30 | R225D |
+| NADZAB     | NAD | 150 | 30 | N/A |
+| MILNE     | MIL | 090 | 30 | R226A |
+| WEDGE 1     | W1 | 285 | 16 | R238 |
+| WEDGE 2     | W2 | 260 | 12 | R238 |
+| WEDGE 3     | W3 | 200 | 16 | R238 |
+
+<figure markdown>
+![Tindal Gates](img/../../aerodromes/classc/img/yptngates.png){width="700"}
+  <figcaption>Tindal Gates</figcaption>
+</figure>
+
+!!! phraseology 
+    CLAS35 has completed operations in R225, and is requesting return to YPTN via TARAKAN Gate.  
+    **TN TCU**: "CLAS35, cleared to YPTN via TARAKAN, thence TN. When ready descend to F130"   
+
+Place/Bearing Distance points in the FDR can be used for routing via the gates, with **TN** as the reference point, in the format PPPPBBBDDD.
+
+!!! example 
+    MOROTAI = TN170030  
```

Comment:
Assume this should be TN27030? Might need to fix the other example on the ADC page